src/ui/server/constants/application/index.ts

```diff
@@ -1,8 +1,12 @@
 import LATEST_VERSION_NUMBER from './versions/latest';
 
+export const SERVICE_NAME = 'EXIP';
+export const INSURANCE = 'Insurance';
+export const QUOTE = 'Quote';
```

Comment:
This might be breaking some things, the moving of `SERVICE_NAME` and `QUOTE` into `application` constants; quote is not part of the application/"insurance"

I think only `INSURANCE` should be in here. In the interests of progressing this PR, to rule out this potentially causing E2E issues, maybe move `SERVICE_NAME` and `QUOTE` back into root constants and we can extract later.